src/pages/individualGallery/components/FormattedLocationWithDialog.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+
+import SinglePointDialog from '../../../components/dialogs/SinglePointDialog';
+import Text from '../../../components/Text';
+
+const buttonAsLinkStyles = {
+  textTransform: 'none',
+  border: 'none',
+  padding: 0,
+  textDecoration: 'underline',
+  cursor: 'pointer',
+  backgroundColor: 'transparent',
+};
+
+export default function FormattedLocationWithDialog({
```

Comment:
what do you think of getting these formattedXYZ components in a subfolder?

Reply:
They are already in a subfolder: `pages/individualGallery/components`. Do you want them further nested within that folder, `pages/individualGallery/components/formatted`? I don't think that is necessary, but I don't mind making the change.

Reply:
oh nope, didn't notice that.